src/chainparams.cpp

```diff
@@ -92,6 +93,11 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_CSV].nStartTime = 1462060800; // May 1st, 2016
         consensus.vDeployments[Consensus::DEPLOYMENT_CSV].nTimeout = 1493596800; // May 1st, 2017
 
+        // Deployment of SegWit.
```

Comment:
add the BIP numbers being deployed.

`Deployment of SegWit, BIP141 and BIP143`